dbt/adapters/redshift/connections.py

```diff
@@ -37,10 +37,24 @@ def get_message(self) -> str:
 logger = AdapterLogger("Redshift")
 
 
+class IdentityCenterTokenType(StrEnum):
+    ACCESS_TOKEN = "ACCESS_TOKEN"
+    EXT_JWT = "EXT_JWT"
+
+
 class RedshiftConnectionMethod(StrEnum):
     DATABASE = "database"
     IAM = "iam"
     IAM_ROLE = "iam_role"
+    IAM_IDENTITY_CENTER_BROWSER = "browser_identity_center"
+
+    @staticmethod
+    def uses_identity_center(method: str) -> bool:
+        return method.endswith("identity_center")
+
+    @staticmethod
+    def is_iam(method: str) -> bool:
```

Comment:
I might be arguing sematics at this point, but I think of this as a class method:
```python
@classmethod
def is_iam(cls, method: str) -> bool:
    return not cls.uses_identity_center(method)
```